systemtests/tests/bconsole-basic/testrunner-status-client

```diff
@@ -38,5 +43,8 @@ run_bconsole
 # if log2.out has not been written, bconsole crashed or disconnected
 # after status client command
 test -f "$tmp/log2.out" || exit 2
+expect_grep "Client \"bareos-fd\" is disabled" \
+	    "$tmp/log3.out" \
+	    "Error missing disabled confirmation."
```

Comment:
that looks good. However, we should probably also check that `log2.out` does *not* contain "Client bareos-fd is disabled", but the proper status output.

Reply:
The reason that the check for log2 is so stupid is that it was like that before. However, it of course makes sense to improve it like you suggested.